gallery/v2_transforms/plot_transforms_v2.py

```diff
@@ -90,7 +90,7 @@
 
 from torchvision import datapoints  # we'll describe this a bit later, bare with us
 
-bboxes = datapoints.BoundingBoxes(
+boxes = datapoints.BoundingBoxes(
```

Comment:
Dataset wrapper uses "boxes" instead of "bboxes". I don't have a pref, but we should align (hence the changes in this example)

Reply:
The reason the dataset wrapper uses `"boxes"` is because our models require that:

https://github.com/pytorch/vision/blob/f7c7bdf5fae5e946e67812c5791d91cec83fccd5/torchvision/models/detection/ssd.py#L333-L341